xml/vm_security.xml

```diff
@@ -152,18 +76,7 @@ Successfully registered system</screen>
     <para>
       You can verify the installation and configuration of the packages.
```

Comment:
There's not much left in this section. Maybe change this paragraph to

You can verify the installation and configuration of the &vmhost; using dmesg or the optional `<command>snphost</command>` tool.

I suppose this suggestion necessitates documenting snphost :-).